front/src/components/mmt-user-menu/mmt-user-menu.component.html

```diff
@@ -3,7 +3,8 @@
 
     <v-list-item-avatar :size="`${$vuetify.breakpoint.smAndDown ? 48 : 130}`" 
                         :class="`${$vuetify.breakpoint.smAndDown ? 'mx-1' : ''}`">
-      <img :src="user.avatar" />
+      <img v-if="user.avatar" :src="user.avatar"  />
+      <v-icon v-else x-large>mdi-camera</v-icon>
```

Comment:
it seems we can create out own component wrapper for avatar with using vuetify avatar https://vuetifyjs.com/en/components/avatars/

Reply:
I've created `mmt-avatar`  component